index.js

```diff
@@ -63,46 +63,56 @@ function message(target, context, message, isBot) {
-    let msgRandom = Math.floor(Math.random() * 5) + 1;
-    let alvoBan = String(mensagem).split(" ");
-    if (alvoBan[1] == undefined) {
-      client.say(
-        target,
-        `Para usar corretamente este comando digite !ban e ao lado o nick da pessoa que vc quer banir! SeemsGood`
-      );
-    } else {
-      switch (msgRandom) {
-        case 1:
-          client.say(
-            target,
-            `/me ${alvoBan[1]} foi banido por falar que php é ruim Kappa`
-          );
-          break;
-        case 2:
+        let randomBan = Math.floor(Math.random() * 10000)
+        let msgRandom = Math.floor(Math.random() * 5) + 1;
+        let alvoBan = String(mensagem).split(" ");
+        if (alvoBan[1] == undefined) {
           client.say(
             target,
-            `/me ${alvoBan[1]} foi banido por não ter feito bootcamp de 2dias de js >( `
+            `Para usar corretamente este comando digite !ban e ao lado o nick da pessoa que vc quer banir! SeemsGood`
           );
-          break;
-        case 3:
-          client.say(target, `/me ${alvoBan[1]} foi banido por usar windows LUL`);
-          break;
-        case 4:
-          client.say(target, `/me ${alvoBan[1]} foi banido por usar js Kappa`);
-          break;
-        case 5:
-          client.say(
-            target,
-            `/me ${alvoBan[1]} foi banido por não deixar no lurk...`
-          );
-          break;
-        default:
-          client.say(
-            target,
-            `/me ${alvoBan[1]} foi banido por falar que java é bom Kappa`
-          );
-      }
-    }
+        } else {
+          switch (msgRandom) {
+            case 1:
+              client.say(
+                target,
+                `/me ${context.username} baniu ${alvoBan[1]} por falar que php é ruim Kappa`
+              );
+              break;
+            case 2:
+              client.say(
+                target,
+                `/me ${context.username} baniu ${alvoBan[1]} por não ter feito bootcamp de 2dias de js >( `
+              );
+              break;
+            case 3:
+              client.say(target, `/me ${context.username} baniu ${alvoBan[1]} por usar windows LUL`);
+              break;
+            case 4:
+              client.say(target, `/me ${context.username} baniu ${alvoBan[1]} por usar js Kappa`);
+              break;
+            case 5:
+              client.say(
+                target,
+                `/me ${context.username} baniu ${alvoBan[1]} por não deixar no lurk...`
+              );
+              break;
+            default:
+              client.say(
+                target,
+                `/me ${context.username} baniu ${alvoBan[1]} por falar que java é bom Kappa`
+              );
+          }
+        }
+        if(randomBan > 100 && randomBan < 1000){
+          client.say(target, `/timeout ${alvoBan[1]} 10`)
+          client.say(target, `${alvoBan[1]} foi banido`)
+        } else if (randomBan > 1000 && randomBan < 7000){
```

Comment:
Acho que a chance ta muito alta? x) Bota `< 3000` pra ficar melhor, se a galera abusar a gente bota mais agressivo hehe
```suggestion
        } else if (randomBan > 1000 && randomBan < 3000){
```